MachUpskillingFY17/JabbR-Core
# Brought over NoteComand

To test: 
So we're having a more general problem with things not displaying in the chatroom, but the note is being changed. If you set a breakpoint in NoteCommand to see if       `context.NotificationService.ChangeNote(callingUser);` runs, you can see that it updates the callingUser's note if you run /note. (running /note with no perams will clear the note).
